src/mcp_atlassian/server.py

```diff
@@ -669,6 +670,33 @@ async def list_tools() -> list[Tool]:
                         "required": ["jql"],
                     },
                 ),
+                Tool(
+                    name="jira_search_fields",
+                    description="Search Jira fields by keyword with fuzzy match",
+                    inputSchema={
+                        "type": "object",
+                        "properties": {
+                            "keyword": {
+                                "type": "string",
+                                "description": "The keyword to search for",
```

Comment:
How about updating the current description when the default value is given?

```suggestion
                                "description": "Keyword for fuzzy search. If left empty, lists the first 'limit' available fields in their default order.",
```